certified-connectors/KyndrylMainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,707 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Kyndryl mainframe connector",
+    "version": "1.0.0",
+    "title": "kyndryl"
+  },
+  "host": "kyndryl-apigateway.azure-api.net",
+  "basePath": "/",
+  "schemes": [
+    "http"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/cb12-policy/getpolicy": {
+      "get": {
+        "tags": [
+          "cb12-policy"
+        ],
+        "operationId": "getPolicy",
+        "parameters": [
+          {
+            "name": "CUSTOMERNUMBER",
```

Comment:
@RTkyndryl for all parameters The 'description' property is required and the 'x-ms-summary' property is required.
Please add both for each parameter in the swagger file. 
```
"name": "CUSTOMERNUMBER",
"description": "Your desctiption",
"x-ms-summary": "Your summary"
```